dsn_test.go

```diff
@@ -582,112 +582,171 @@ func TestParseDSN(t *testing.T) {
 		},
 	}
 
+	for _, at := range []AuthType{AuthTypeExternalBrowser, AuthTypeOAuth} {
+		testcases = append(testcases, tcParseDSN{
+			dsn: fmt.Sprintf("@host:777/db/schema?account=ac&protocol=http&authenticator=%v", strings.ToLower(at.String())),
+			config: &Config{
+				Account: "ac", User: "", Password: "",
+				Protocol: "http", Host: "host", Port: 123,
```

Comment:
is it ok that port is different than in dsn? also the same in other tests?

Reply:
It doesn't matter, because earlier check fails. But to have order I fixed it. Thanks!